extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -441,7 +442,8 @@ export class AmpAutocomplete extends AMP.BaseElement {
   generateSrc_(opt_query = '') {
     const encodedQueryKey = encodeURIComponent(this.queryKey_);
     const encodedQuery = encodeURIComponent(opt_query);
-    return `${this.srcBase_}?${encodedQueryKey}=${encodedQuery}`;
+
+    return addParamToUrl(this.srcBase_, encodedQueryKey, encodedQuery);
```

Comment:
`addParamToUrl` calls `encodeURIComponent` on the query key and value, so the two lines above can be removed also.